packages/nocodb/src/helpers/getAst.ts

```diff
@@ -18,7 +18,7 @@ const getAst = async ({
     nested: { ...(query?.nested || {}) },
     fieldsSet: new Set(),
   },
-  getHiddenColumn = false,
+  getHiddenColumn = query?.['getHiddenColumn'],
```

Comment:
`query?.['getHiddenColumn'] || false` maybe?

Reply:
`undefined` or `null` will be falsey value so without `|| false` also fine